rules-tests/Renaming/Rector/Name/RenameClassRector/Fixture/rename_class.php.inc

```diff
@@ -22,7 +22,7 @@ namespace MyNewNamespace;
 class MyNewClass
 {
     /**
-     * @return \MyNewNamespace\MyNewClass
+     * @return MyClass
```

Comment:
this should be kept as `MyClass` changed to `MyNewClass`

Reply:
Not sure why this one keeps changing :/ It has somethign to do with cache, possibly similar method info.